xen-ocaml/bindings/main.c

```diff
@@ -26,6 +27,11 @@ int errno;
 static char *argv[] = { "mirage", NULL };
 static unsigned long irqflags;
 
+/* XXX TODO: keep in sync with mirage-xen-minios */
+void setup_xen_features(void);
+void init_console(void);
+void init_gnttab(void);
+
```

Comment:
partially fixed in 63eee886fcb9789f6d9c38acd9702c19f2e5b9fc -- `setup_xen_features` isn't exposed in any headers, just the other two.